content/docs/command-reference/import-url.md

```diff
@@ -37,6 +39,12 @@ external data source changes. Example scenarios:
 having to manually copy files from the supported locations (listed below), which
 may require installing a different tool for each type.
 
+When you don't want to store the target data in your local system, you can still
+create an import `.dvc` file while transferring a file or directory directly to
+remote storage, by using the `--to-remote` option. See the
+[Import straight to remote](#example-transfer-to-remote-storage) example for
+more details.
```

Comment:
Can we put something like this in the `add` reference Description (instead of the **Transferring data directly to remote storage** section) ?